ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml

```diff
@@ -344,6 +344,26 @@ StatusSectionLayout {
                         mintPanel.errorText = qsTr("Unknown error")
                         mintPanel.isFeeLoading = true
                     }
+                    function onDeploymentStateChanged(status, url) {
+                        let title
+                        let loading = false
+                        let type = Constants.ephemeralNotificationType.normal
+                        if (status === Constants.DeployState.InProgress) {
```

Comment:
```
switch (status) {
case Constants.DeployState.InProgress: {
...
```
May looks better :)

Reply:
Also, even with the `if`, you need a default for the title, otherwise, it would throw.